main.go

```diff
@@ -851,8 +873,11 @@ func writeClusterHeader(w *tabwriter.Writer) (err error) {
 		"AutoScale",
 		"Status",
 	}
-	s := strings.Join(headerFields, "\t")
+	return writeRow(w, headerFields)
+}
 
+func writeRow(w *tabwriter.Writer, fields []string) (err error) {
+	s := strings.Join(fields, "\t")
```

Comment:
Yup, this looks pretty good. I figured you would need to do this with our first new set of rows.